mirdata/ikala.py

```diff
@@ -41,8 +38,22 @@
 )
 
 
-def download(data_home=None):
-    save_path = get_save_path(data_home)
+def download(data_home=None, clobber=False):
+    save_path = utils.get_save_path(data_home)
+    dataset_path = os.path.join(save_path, IKALA_DIR)
+
+    if clobber:
+        utils.clobber_all(IKALA_METADATA,
+                          dataset_path,
+                          data_home)
+    if utils.check_validated(dataset_path):
+        print("""
+                The {} dataset has already been validated.
+                If you feel this is a mistake please rerun and set clobber to true.
+                """.format(IKALA_DIR))
+        return
+
+    validate(dataset_path, data_home)
     print("""
```

Comment:
This should only print if validate "fails".

Reply:
Same comment for the other loaders.

Reply:
updated.